test/scripts/setup-konvoy.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
Where is this script used? I think it'd be helpful to have a comment here.

Reply:
```suggestion
#!/bin/bash
# NOTE: used by teamcity
```